xplique/attributions/image_object_detector_explainer.py

```diff
@@ -0,0 +1,280 @@
+"""
+Module related to Object detector method
+"""
+
+from typing import Iterable, Tuple, Union, Optional
+import abc
+
+import tensorflow as tf
+import numpy as np
+
+from xplique.attributions.base import BlackBoxExplainer
+from xplique.commons.tf_operations import inference_batching
+
+
+class IouCalculator:
+    """
+    Used to compute the Intersection Over Union.
+    """
+    @abc.abstractmethod
+    def intersect(self, obj1: tf.Tensor, obj2: tf.Tensor) -> tf.Tensor:
+        """
+        Compute the intersection between two batched objects
+        An object could be box, segmentation mask...
+
+        Parameters
+        ----------
+        obj1

```

Comment:
better use pythonic variable name such as obj_a or obj_1 if you want. (it's minor just for the next PRs)